test/test_log_prob.jl

```diff
@@ -0,0 +1,118 @@
+using Celeste: Model, SensitiveFloats, ElboDeriv, Infer
+using Base.Test
+"""
+test log_prob.jl and log_prob_util.jl
+"""
+
+####################
+# Helper functions #
+####################
+function true_star_init()
```

Comment:
Let's not duplicate code.  You can put these helper functions in ```DerivativeTestUtils``` and maybe call it ```TestUtils``` instead if you like.